Boards.h

```diff
@@ -299,6 +299,22 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 #define PIN_TO_PWM(p)           PIN_TO_DIGITAL(p)
 #define PIN_TO_SERVO(p)         (p) // deprecated since v2.4
 
+// Arduino Primo
+#elif defined(NRF52)
```

Comment:
Use a more specific name than NRF52, otherwise this block may be defined for other NRF52-based boards in the future. I assume there is a define like "ARDUINO_<ARCH>_PRIMO" for this board.